src/transac/components/alerts/alerts.module.coffee

```diff
@@ -0,0 +1,23 @@
+###
+#   @desc Components for providing Alert messaging for the application, via toastr notifications.
+###
+angular.module('transac.alerts',
+  [
+    # deps
+    'toastr'
+    'ngAnimate'
+  ])
+  .config((toastrConfig)->
+    toastrConfig.timeOut = 1500
+    toastrConfig.positionClass = 'toast-top-right'
+    toastrConfig.preventDuplicates = false
+    toastrConfig.progressBar = true
+  )
+  .constant('TOASTR_OPTS',
+    success:
+      timeOut: 800
+    info:
+      timeOut: 800
+    error:
+      timeOut: 1500
```

Comment:
Yes that's by design, notifications are sent when you successfully allow / deny transactions, and you can do this very quickly, so long durations would cause the notifications to build up very fast. The error timeOut is longer - and could be longer if preferred.